README.md

```diff
@@ -1 +1,25 @@
-# couch-tracker-documentation
\ No newline at end of file
+# Couch-Tracker
+
+An Android application, to keep track of the TV shows you follow, and the movies you've watched.
+
+This repository contains all the documents and design decisions we've made.
+
+
+## Tenets
+
+- Open-source
```

Comment:
Why no more free? 😂

Reply:
Well, the "no paywalling for features done locally and don't require an ongoing cost from us" kind of covers that.

Also, if we want to keep the door open for charging users when/if we build a managed cloud service, I'd rather be upfront now.

Reply:
ah ok, makes sense